docs/user/filenodes.md

```diff
@@ -0,0 +1,33 @@
+# FlowForge File Nodes
+
+Projects running within FlowForge include a set of nodes that make it possible
+in a containered environment to read and write to persistent storage.
+
+The File Nodes are a clone of the original Node-RED nodes so that flows written
+for the cloud can be ran the same on a device which uses its own local filesystem.
+
+For that reason, these nodes cannot be used at the same time as the built-in 
+Node-RED file nodes. To use these nodes, you must exclude 10-file-js in the 
+`nodesExcludes` array in your settings file.
```

Comment:
I don't have access to these settings? How does an end user exclude anything?

Reply:
outdated